From: Jesper Juhl <jj@chaosbits.net>
Date: Sun, 5 Feb 2012 20:11:27 +0100
Subject: NVM Express: Remove unneeded include of linux/version.h from nvme.c
Git-commit: d0156f4d62f1c6b07ea6ffd32b10100d09e25d83
References: FATE#313627
Patch-Mainline: v3.4

There's no need for drivers/block/nvme.c to include linux/version.h,
so remove the include.

Signed-off-by: Jesper Juhl <jj@chaosbits.net>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/block/nvme.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/block/nvme.c b/drivers/block/nvme.c
index f4996b0..e063248 100644
--- a/drivers/block/nvme.c
+++ b/drivers/block/nvme.c
@@ -39,7 +39,6 @@
 #include <linux/sched.h>
 #include <linux/slab.h>
 #include <linux/types.h>
-#include <linux/version.h>
 
 #define NVME_Q_DEPTH 1024
 #define SQ_SIZE(depth)		(depth * sizeof(struct nvme_command))
-- 
1.7.4.2

